src/utils/api/client.ts

```diff
@@ -249,14 +250,30 @@ export async function searchDiscussions(
   return apiRequestAuth(Constants.GITHUB_API_GRAPHQL_URL, 'POST', token, {
     query: print(QUERY_SEARCH_DISCUSSIONS),
     variables: {
-      queryStatement: formatSearchQueryString(
+      queryStatement: formatAsGitHubSearchSyntax(
         notification.repository.full_name,
         notification.subject.title,
-        notification.updated_at,
       ),
-      firstDiscussions: 10,
+      firstDiscussions: 1,
```

Comment:
What's the impact of this?

Reply:
Nevermind, got it, we were never looking at the rest of the comments, only to the latest